### ISSUE TITLE
Laryssa spawned double

### ISSUE THREAD

**Issue by MikeX1234** [Gitjas/Brages_Redemption#2]
In my game Laryssa, cousin of Brage, spawned double.
I found and killed Borda before I found and rescued Brage.
In may case the culprit was in this section of the area script, which was extended by q13_4800.baf:

```
/* [Borda is still alive and has to appear a second time.] Laryssa spawns outside temple in Nashkel after Nalin looked at the sword */
IF
	Global("C#q13_LarissaSpawn","MYAREA",0)
	OR(2)
		Global("C#q13Brage_FakeMerchant","GLOBAL",0)
		Global("C#q13Brage_FakeMerchant","GLOBAL",1)
	!Dead("Brage")
	GlobalGT("C#q13Brage_NalinVouch","GLOBAL",1)
	OR(2)
		GlobalLT("C#Q13_BragesSwordQuest","GLOBAL",4)
		Global("C#Q13_BragesSwordQuest","GLOBAL",6)
THEN
	RESPONSE #100
		SetGlobal("C#q13_LarissaSpawn","MYAREA",1)
		CreateCreature("c#q13lar",[2450.1198],13)
		IncrementGlobal("C#q13Brage_FakeMerchant","GLOBAL",2)
		Continue()
END

/* in case innocence was proven with third letter out of Tazok's tent (PC didn't need Larissa to find the merchant */
IF
	Global("C#q13_LarissaSpawn","MYAREA",0)
	Dead("BORDA") 
	!Dead("Brage")
	GlobalGT("C#q13Brage_NalinVouch","GLOBAL",1)
	OR(2)
		GlobalLT("C#Q13_BragesSwordQuest","GLOBAL",4)
		Global("C#Q13_BragesSwordQuest","GLOBAL",6)
THEN
	RESPONSE #100
		SetGlobal("C#q13_LarissaSpawn","MYAREA",1)
		CreateCreature("c#q13lar",[2450.1198],13)
		Continue()
END
````
One of the two sections might miss conditions, so that they don't come true at the same evaluation cycle.
If not, the same functionality w/o double spawning can be reached with only one section like this:
```
/* [Borda is still alive and has to appear a second time.] Laryssa spawns outside temple in Nashkel after Nalin looked at the sword */
/* OR */
/* in case innocence was proven with third letter out of Tazok's tent (PC didn't need Larissa to find the merchant */
IF
	Global("C#q13_LarissaSpawn","MYAREA",0)
	OR(3)
		Dead("BORDA") 
		Global("C#q13Brage_FakeMerchant","GLOBAL",0)
		Global("C#q13Brage_FakeMerchant","GLOBAL",1)
	!Dead("Brage")
	GlobalGT("C#q13Brage_NalinVouch","GLOBAL",1)
	OR(2)
		GlobalLT("C#Q13_BragesSwordQuest","GLOBAL",4)
		Global("C#Q13_BragesSwordQuest","GLOBAL",6)
THEN
	RESPONSE #100
		SetGlobal("C#q13_LarissaSpawn","MYAREA",1)
		CreateCreature("c#q13lar",[2450.1198],13)
		IncrementGlobal("C#q13Brage_FakeMerchant","GLOBAL",2)
		Continue()
END
```
This worked in my game.

However, the esieast solution might be to just remove the continue() in the first section...

**Comment by MikeX1234** [comment]
Tested again with just removing continue() from the first of the two blocks, works as well...
@Gitjas

**Comment by Gitjas** [comment]
Yes, removing the Continue() makes a lot of sense. Thank you for the report!
And I totally missed this EDIT this report before today. -.-

**Comment by Gitjas** [comment]
(THe Continue() is there for other reasons, though, so I'll look for another fix to this.)

**Comment by Gitjas** [comment]
Fixed it by adding !Dead("BORDA") check to first scriptblock. Will be fixed in next version.

**Comment by Gitjas** [comment]
Was fixed in v7.2.
